test/fakes/fakeentryclient/entryclient.go

```diff
@@ -49,7 +49,7 @@ func New(t *testing.T, trustDomain spiffeid.TrustDomain, ds datastore.DataStore,
 	service := entry.New(entry.Config{
 		TrustDomain: trustDomain,
 		DataStore:   ds,
-		//EntryFetcher: authorizedEntryFetcherWithCache,
+		// EntryFetcher: authorizedEntryFetcherWithCache,
```

Comment:
is this required? can you remove this comment instead?

Reply:
Not sure why this was commented out in the first place. I think my IDE formatting settings (`gofumpt`) made this change on my behalf, but I can remove the commented out line.